Civixero.php

```diff
@@ -152,7 +152,7 @@ function civixero_civicrm_navigationMenu(&$menu) {
   ]);
   $connectors = _civixero_get_connectors();
   foreach ($connectors as $connectorID => $details) {
-    _Civixero_civix_insert_navigation_menu($menu, 'Administer/legacyxero/', [
+    _Civixero_civix_insert_navigation_menu($menu, 'Administer/CiviContribute/Xero/', [
```

Comment:
Oh - I just moved that one to the mgd file - now all the ones we want to keep are in the mgd file